fn_ioc_parser/fn_ioc_parser/util/ioc_parser_helper.py

```diff
@@ -0,0 +1,152 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+import os
+import tempfile
+from docx import Document
+import io
+from pdfminer.converter import TextConverter
+from pdfminer.pdfinterp import PDFPageInterpreter
+from pdfminer.pdfinterp import PDFResourceManager
+from pdfminer.pdfpage import PDFPage
+import xlrd
+
+
+class IOCParserHelper(object):
+    def __init__(self):
+        # Resilient Rest Api calls url's
+        self.ARTIFACT_META_DATA_URL = "/incidents/{0}/artifacts/{1}"
+        self.ARTIFACT_DATA_URL = "/incidents/{0}/artifacts/{1}/contents"
+        self.ATTACHMENT_META_DATA_URL = "/incidents/{0}/attachments/{1}"
+        self.ATTACHMENT_DATA_URL = "/incidents/{0}/attachments/{1}/contents"
+        self.TASK_META_DATA_URL = "/tasks/{0}/attachments/{1}"
+        self.TASK_DATA_URL = "/tasks/{0}/attachments/{1}/contents"
+
+    @classmethod
+    def extract_text_from_bytes_data(cls, filename, file_data_bytes):
+        if filename.find('.pdf') != -1:
+            """
+            Converting .pdf file data to plain text format
+            """
+            file_string_data = IOCParserHelper.extract_text_from_pdf(file_data_bytes)
+        elif filename.find('.docx') != -1:
+            """
+            Converting .docx file data to plain text format
+            """
+            file_string_data = IOCParserHelper.extract_text_from_docx(file_data_bytes)
+        elif filename.find('.xls') != -1 or filename.find('.xlsx') != -1:
+            """
+            Converting .xls/.xlsx file data to plain text format
+            """
+            file_string_data = IOCParserHelper.extract_text_from_xls_xlsx(file_data_bytes)
+        elif filename.split('.')[-1].strip() in ['doc', 'odt', 'ott', 'dot']:
+            file_string_data = None
+        else:
+            file_string_data = file_data_bytes
+
+        if file_string_data:
+            file_string_data = str(file_string_data)
```

Comment:
* Is this supported in Python2.7 and Python3.6?
* Has it been tested if `file_string_data` contains some Unicode characters?